### PR TITLE
:sparkles: feature: Add root compute workspace

### PR DESCRIPTION
**Pull request by qiujian16**
Signed-off-by: Jian Qiu <jqiu@redhat.com>

## Summary
create root:compute workspace, apiexport and apiresourceschemas.

1. includes apiresourceschema of deployments/services/ingreses
2. `root-compute-workspace` battery flag added

## Related issue(s)

Fixes https://github.com/kcp-dev/kcp/issues/1959

### CODE REVIEW

**Review by stevekuznetsov — COMMENTED** [review]
I would really rather see a resource-yaml initializer and not more of these bootstrap controllers.

**Comment by ncdc** [comment]
@stevekuznetsov would you still compile the yaml into an embedded filesystem?

**Comment by stevekuznetsov** [comment]
Likely no. Would love to see a config mechanism at runtime so everyone could use it, including our bootstrap. None of this is really crucial to kcp core coming up, right?

**Comment by stevekuznetsov** [comment]
Don't need to block this if it's "urgent" but we need to stop proliferating these controllers at some point, yes?

**Comment by ncdc** [comment]
Thanks. Maybe let's file an issue for that?

**Comment by davidfestal** [comment]
> None of this is really crucial to kcp core coming up, right?

Wasn't this the purpose of the batteries concept introduced by @sttts ?
cf @sttts comment on this related issue: https://github.com/kcp-dev/kcp/issues/1959#issuecomment-1245879192

**Comment by sttts** [comment]
In a production setup this will be done by some Gitops agent. What we put into `kcp start` is a toy (a battery). Let's not overdesign this. A simple bootstrap controller by far enough.

**Comment by sttts** [comment]
Sgtm. Giving some more eyes a chance to look over this, maybe @jmprusi.

/approve

File: config/rootcompute/clusterworkspace-compute.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: tenancy.kcp.dev/v1alpha1
+kind: ClusterWorkspace
+metadata:
+  name: compute
+  annotations:
+    bootstrap.kcp.dev/create-only: "true"
+    bootstrap.kcp.dev/battery: root-compute-workspace
+spec:
+  type:
+    name: organization
```
Comment (sttts) [inline config/rootcompute/clusterworkspace-compute.yaml:10]: maybe universal?

**Pull request merged**